x/trigger/types/keys.go

```diff
@@ -0,0 +1,153 @@
+package types
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	fmt "fmt"
+	"strings"
+)
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "trigger"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for slashing
+	RouterKey = ModuleName
+
+	// QuerierRoute defines the module's query routing key
+	QuerierRoute = ModuleName
+
+	TriggerIDLength  = 8
+	QueueIndexLength = 8
+	GasLimitLength   = 8
+)
+
+var (
+	TriggerKeyPrefix       = []byte{0x01}
+	EventListenerKeyPrefix = []byte{0x02}
+	QueueKeyPrefix         = []byte{0x03}
+	GasLimitKeyPrefix      = []byte{0x04}
+	NextTriggerIDKey       = []byte{0x05}
+	QueueStartIndexKey     = []byte{0x06}
+	QueueLengthKey         = []byte{0x07}
+)
+
+// GetEventListenerKey converts an event name and trigger ID into an event registry key format.
+func GetEventListenerKey(eventName string, id TriggerID) []byte {
+	eventNameBytes := GetEventNameBytes(eventName)
+
+	triggerIDBytes := make([]byte, TriggerIDLength)
+	binary.BigEndian.PutUint64(triggerIDBytes, id)
+
+	key := EventListenerKeyPrefix
+	key = append(key, eventNameBytes...)
+	key = append(key, triggerIDBytes...)
+	return key
```

Comment:
```suggestion
	triggerIDBytes := make([]byte, TriggerIDLength)
	binary.BigEndian.PutUint64(triggerIDBytes, id)

	key := GetEventListenerPrefix(eventName)
	key = append(key, triggerIDBytes...)
	return key
```

Reply:
Great cleanup, and it looks a lot nicer.